podpac/core/coordinates/cfunctions.py

```diff
@@ -36,18 +36,18 @@ def crange(start, stop, step, name=None):
 
 def clinspace(start, stop, size, name=None):
```

Comment:
I think the stacked linspace is important, and I think we can use the name argument here to support it. Something like this

```
    ...
    dims = name.split('_')
    if np.array(start).size != len(dims):
        raise size mismatch

    a = np.array([start, stop])
    if a.ndim == 2:
        c = StackedCoordinates([UniformCoordinates1d(start[i], stop[i], size=size) for i, dim in enumerate(dims)])
    else:
        c = UniformCoordinates1d(start, stop, size=size, name=name)
    return c
```